webapp-django/crashstats/crashstats/models.py

```diff
@@ -1130,6 +1131,16 @@ class ProductBuildTypes(SocorroMiddleware):
     )
 
 
+class ProductIDMap(SocorroMiddleware):
```

Comment:
If I were writing this PR, I'd have made this `ProductProductIDMap`. I see how it looks silly, though.